lib/miq_automation_engine/service_models/miq_ae_service_iso_datastore.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceIsoDatastore < MiqAeServiceModelBase
+    expose :iso_images, :association => true
```

Comment:
@matobet This PR looks good, just wondering if it makes sense to add an `expose` for the `ext_management_system`.  Do you think it would be useful?

https://github.com/ManageIQ/manageiq/blob/master/app/models/iso_datastore.rb#L2